satellite-imagery.md

```diff
@@ -0,0 +1,136 @@
+for observing the earth in different ways and each instrument usually measures
+different channels or wavelengths of electromagnetic radiation emitted from
+or reflected off the earth. The files that these data are provided in depend
+greatly on what the instrument is measuring, which organization is running the
+satellite mission, who is creating the files, and even what command line flags
+were used when creating the files. This makes loading data from these files
+very difficult.
+
+Loading data from files is just the beginning of a typical satellite
+processing workflow. The data may need to be calibrate or corrected to remove
+certain affects from the instrument, sun, or atmosphere and be more useful for
+a particular type of analysis. To be viewable in various visualization tools
+the data must also be projected or resampled to a certain model and shape of
+the earth. To be viewed as images the data is usually scaled in specific ways
+to bring out certain features of the data including making RGB color images by
+combining multiple channels of the data. Lastly, the data usually has to be
+written to another file format to be used by a scientist's target audience
+whether it be other scientists, forecasters, or the general public. Each group
+has certain tools and certain expectations about what they should be able to
+do with the data. As satellite instrument technology advances scientists are
+now having to learn how to handle more channels for each instrument and at
+spatial and temporal resolutions that were unheard of when they were learning
+how to use satellite data. If they are lucky scientists may have access to a
+high performance computing system, while the rest may have to settle for long
+execution times on their desktop or laptop machines.
+
+SatPy tries to make all of this processing easy to handle with simple
+interfaces and logical defaults to produce the best results. By optimizing the
+parts of the processing that typically take a lot of time and memory it is our
+hope that scientists can worry about the science and leave the annoying parts
+to SatPy.
+
+
+How Dask helps
+--------------
+
+SatPy was originally drawn to dask for its lazy evaluation and multi-threaded
+processing. Dask is used as the array backend for the xarray library's
+`DataArray` object which is used as the main data container object in SatPy.
+DataArray objects allow SatPy to keep the satellite imagery data, data
+coordinates, and other metadata all in one place. Before using xarray and dask
+SatPy used numpy masked arrays but as new satellite instruments were launched
+our users simply couldn't handle the amount of memory required to perform
+basic operations. The processing involved in SatPy requires a lot of
+intermediate variables/calculations and comparisons between different data
+arrays. With masked arrays and the right calculations you can quickly run out
+of memory on a 16-32GB machine with instruments launched in the last 5-10
+years.
```

Comment:
I think that some of this could be in the section above.  

> My group maintains a library, SatPy, to manage these situations.  Historically we used Numpy masked arrays, but with new satellite instruments our users' machines couldn't handle the amount of memory required to perform basic operations ...

Where "the problem you're trying to solve" seems to be "scaling satellite image processing pipelines"